packages/api/tests/lib/rules/test-createRuleResources.js

```diff
@@ -0,0 +1,358 @@
+  await Promise.all([
+    putJsonS3Object(
+      process.env.system_bucket,
+      workflowfile,
+      {}
+    ),
+    putJsonS3Object(
+      process.env.system_bucket,
+      templateFile,
+      {}
+    ),
+  ]);
+});
+
+test.beforeEach(async () => {
+  await deleteKinesisEventSourceMappings();
+});
+
+test.after.always(async (t) => {
+  // cleanup table
+  await recursivelyDeleteS3Bucket(process.env.system_bucket);
+  delete process.env.system_bucket;
+  delete process.env.stackName;
+  await destroyLocalTestDb({
+    knex: t.context.testKnex,
+    knexAdmin: t.context.testKnexAdmin,
+    testDbName,
+  });
+});
+
+test('Creating a rule trigger defaults enabled to true', async (t) => {
+  const rule = fakeRuleRecordFactory({
+    type: 'onetime',
+    workflow,
+    enabled: true,
+  });
+  // remove enabled from rule to be created
+  delete rule.enabled;
+
+  // create rule trigger
+  const ruleWithTrigger = await createRuleTrigger(rule);
+
+  t.true(ruleWithTrigger.enabled);
+});
+
+test('Creating an invalid rule does not create workflow triggers', async (t) => {
+  const rule = fakeRuleRecordFactory({
+    type: 'onetime',
+    workflow,
+    enabled: true,
+  });
+  rule.type = 'invalid';
+
+  await t.throwsAsync(
+    () => createRuleTrigger(rule),
+    { name: 'ValidationError' }
+  );
+});
+
+test('Creating rule triggers for a kinesis type rule adds event mappings', async (t) => {
+  const kinesisRule = fakeRuleRecordFactory({
+    type: 'kinesis',
+    workflow,
+    enabled: true,
+    value: randomString(),
+  });
+  // create rule
+  const createdRule = await createRuleTrigger(kinesisRule);
+  const kinesisEventMappings = await getKinesisEventMappings();
+  const consumerEventMappings = kinesisEventMappings[0].EventSourceMappings;
+  const logEventMappings = kinesisEventMappings[1].EventSourceMappings;
+
+  t.is(consumerEventMappings.length, 1);
+  t.is(logEventMappings.length, 1);
+  t.is(consumerEventMappings[0].UUID, createdRule.arn);
+  t.is(logEventMappings[0].UUID, createdRule.log_event_arn);
+
+  t.is(createdRule.name, kinesisRule.name);
+  t.is(createdRule.value, kinesisRule.value);
+  t.false(createdRule.arn === undefined);
+  t.false(createdRule.log_event_arn === undefined);
+
+  // Clean Up
+  // await deleteRuleResources(t.context.testKnex, kinesisRule);
+  await deleteKinesisEventSourceMappings();
+});
+
+test('Creating an invalid kinesis type rule does not add event mappings', async (t) => {
+  const kinesisRule = fakeRuleRecordFactory({
+    type: 'kinesis',
+    workflow,
+    enabled: true,
+    value: randomString(),
+  });
+  delete kinesisRule.name;
+  await t.throwsAsync(createRuleTrigger(kinesisRule), { name: 'SchemaValidationError' });
+
+  const kinesisEventMappings = await getKinesisEventMappings();
+  const consumerEventMappings = kinesisEventMappings[0].EventSourceMappings;
+  const logEventMappings = kinesisEventMappings[1].EventSourceMappings;
+
+  console.log(JSON.stringify(kinesisEventMappings));
```

Comment:
Remove this

Reply:
[`907d615` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/907d6158686dd2f2b5bc640e15c153114fce40ae)

Reply:
[`6e4273e` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/6e4273e419e493d91939e1f3fc8ea21e59aa6d22)